middleware/lib/sfMiddleware.ts

```diff
@@ -193,14 +206,14 @@ export function applySfFnMiddleware(cloudEvent: CloudEvent, headers: ReadonlyMap
     if (!data) {
         throw new Error('Data field of the cloudEvent not provided in the request');
     }
-    const ceCtx = isSpec1 ? decodeCeAttrib(cloudEvent.getExtensions()['sfcontext']) : data.context;
-    const ceFnCtx = isSpec1 ? decodeCeAttrib(cloudEvent.getExtensions()['sffncontext']) : data.sfContext;
+    const ceCtx = isSpec1 ? decodeCeAttrib(cloudEvent['sfcontext']) : data['context'];
```

Comment:
What's the reasoning for moving from `.context`? Just curious.

Reply:
The main goal was to avoid de-serializing customer payload in the proxy or outer layers of function processing.  If invoked with HttpBinary encoding, we would only need to look at http headers to be able to access or update the salesforce/function context objects from cloudevent extensions.  The prior implementation mixed customer payload with salesforce/function contexts.